api/helpers/game-states/pack-gamestate.js

```diff
@@ -74,6 +74,9 @@ module.exports = {
 
       const combinedData = { ...gameState, ...convertedData };
 
+      // Remove the id field to prevent database duplicate id errors
+      delete combinedData.id;
```

Comment:
Need to delete the `id` from the unpacked GameState object so that the backend doesn't attempt to write the id field into the newly inserted GameStateRow when saving